front-end/src/lib/URL.js

```diff
@@ -0,0 +1,5 @@
+const URL = {
+  getLoans: 'http://10.22.18.66:3000/loans'
```

Comment:
@JialinWang1 why is this in a `lib` folder?

Reply:
also extract the `baseUrl` into a constant and use this as a default value, but override it with `BASE_URL` env var